cea/globalvar.py

```diff
@@ -501,6 +501,23 @@ def __init__(self):
         self.initial_temp_air_prev = 21
         self.initial_temp_m_prev = 16
 
+        # ==============================================================================================================
+        # Columns to write for the demand calculation
+        # ==============================================================================================================
+        self.demand_totals_csv_columns = ['Name', 'Af_m2', 'Aroof_m2', 'GFA_m2', 'occ_pax', 'QHf_MWhyr', 'QCf_MWhyr',
+                                          'Ef_MWhyr', 'Qhsf0_kW', 'Qhsf_MWhyr', 'Qhs_MWhyr', 'Qwwf0_kW', 'Qwwf_MWhyr',
+                                          'Qww_MWhyr', 'Qhprof_MWhyr', 'Qcsf0_kW', 'Qcsf_MWhyr', 'Qcs_MWhyr',
+                                          'Qcref_MWhyr', 'Qcdataf_MWhyr', 'Ealf0_kW', 'Ealf_MWhyr', 'Eauxf_MWhyr',
+                                          'Ecaf_MWhyr', 'Edataf_MWhyr', 'Eprof_MWhyr', 'Tshs0_C', 'Trhs0_C', 'Tscs0_C',
+                                          'Trcs0_C', 'Tsww0_C', 'Vw_m3yr', 'Vww0_m3', 'Vww_m3yr', 'mcphs0_kWC',
+                                          'mcpww0_kWC', 'mcpcs0_kWC']
+        self.demand_building_csv_columns = ['DATE', 'Name', 'occ_pax', 'QHf_kWh', 'QCf_kWh', 'Ef_kWh', 'Qhsf_kWh',
+                                            'Qhs_kWh', 'Qhprof_kWh', 'Qwwf_kWh', 'Qww_kWh', 'Qww_tankloss_kWh',
+                                            'Qcsf_kWh', 'Qcs_kWh', 'Qcref_kWh', 'Qcdataf_kWh', 'Ealf_kWh', 'Eauxf_kWh',
+                                            'Ecaf_kWh', 'Edataf_kWh', 'Epro_kWh', 'Tshs_C', 'Trhs_C', 'Tscs_C',
+                                            'Trcs_C', 'Tsww_C', 'Trww_C', 'Tww_tank_C', 'Tsref_C', 'Trref_C',
```

Comment:
The value Tww_tank_C is not in the master branch and it is not necessary. I might be missing more.

When including or excluding any variable from the output files (which happens often), should both lists be changed accordingly (one in globalvar.py and one in demand.py)? if so, i believe increasing the complexity of such a task should be left for later (when we have a better product). 

The order of columns seems to be a low priority functionality, and it might instead add up to the chances of making mistakes here and there while coding. I recall the phrase of daren: "early optimization is the root of evil". are we facing one of these cases in here? Maybe i am missing something. @daren-thomas would this be the case?

@amraladdin @martin-mosteiro, would this functionality overcome the drawbacks?